packages/snaps-cli/src/cmds/build/bundleUtils.ts

```diff
@@ -128,3 +129,50 @@ export function postProcess(
 
   return processedString;
 }
+
+/**
+ * Processes dependencies and updates argv with an options object
+ * @param argv
+ */
+export function processDependencies(argv: YargsArgs) {
+  const { depsToTranspile, transpilationMode } = argv;
+  const babelifyOptions: Record<string, any> = {};
+  if (transpilationMode === TranspilationModes.localAndDeps) {
+    const regexpStr = getDependencyRegExp(depsToTranspile as string[]);
+    if (regexpStr !== null) {
+      babelifyOptions.ignore = [regexpStr];
+    }
+  }
+  return babelifyOptions;
+}
+
+/**
+ * Processes a string of space delimited dependencies into one regex string
+ * @param dependencies
+ * @returns a regexp string
+ */
+export function getDependencyRegExp(dependencies: string[]): RegExp | null {
+  let regexp: string | null = null;
+  if (!dependencies || dependencies.includes('.') || !dependencies.length) {
+    return /\/node_modules\/(?!.+)/u;
+  }
+  const paths: string[] = sanitizeDependencyPaths(dependencies);
+  regexp = `/node_modules/(?!${paths.shift()}`;
+  paths.forEach((path) => (regexp += `|${path}`));
+  regexp += '/)';
+  return RegExp(regexp, 'u');
+}
+
+/**
+ * Helper function remove any leading and trailing slashes from dependency list
+ * @param dependencies
+ * @returns an array of sanitized paths
+ */
+export function sanitizeDependencyPaths(dependencies: string[]): string[] {
+  if (!dependencies.length) {
+    return ['.'];
+  }
```

Comment:
I think this is redundant because of how this is used. Up to you whether to keep it.

Reply:
yes! it is redundant, removed!